Configurations/10-main.conf

```diff
@@ -1165,6 +1165,7 @@ my %targets = (
         module_ldflags   => "-Wl,-G,-bsymbolic,-bnoentry",
         shared_ldflag    => "-Wl,-G,-bsymbolic,-bnoentry",
         shared_defflag   => "-Wl,-bE:",
+        shared_fipsflag  => "-Wl,-binitfini:init:cleanup,-bE:",
```

Comment:
Is the order important?

Reply:
Let me clarify, 'cause I can see that as this argument is constructed, of course order is important.  What I wonder is, can `-Wl,-binitfini:init:cleanup` be separate from `-Wl,-bE:{file}`, and does it matter if the `-binitfini:` option comes before or after the `-bE:` option?

Reply:
It can be separate and the order is not important...